src/Mgate/PubliBundle/Controller/DocumentController.php

```diff
@@ -65,24 +66,18 @@ public function voirAction(Document $documentType)
 
     /**
      * @Security("has_role('ROLE_SUIVEUR')")
+     * @param Request $request
+     * @param Etude $etude
+     * @return bool|\Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
```

Comment:
`RedirectResponse` is unnecessary as already included by  `Response` and unless I'm missing something `bool` is never returned so can be removed and a `Response` return type can be added